### PR TITLE
Migrate tests from "store/storetest/command_webhook_store.go" to use testify

### PR DESCRIPTION
**Pull request by gupsho**
#### Summary
Migrate tests from "store/storetest/command_webhook_store.go" to use testify

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/12766

### CODE REVIEW

File: store/storetest/command_webhook_store.go [hunk 1]
```diff
@@ -49,25 +43,22 @@ func testCommandWebhookStore(t *testing.T, ss store.Store) {
 	h2, err = cws.Save(h2)
 	require.Nil(t, err)
 
-	if _, err := cws.Get(h2.Id); err == nil || err.StatusCode != http.StatusNotFound {
-		t.Fatal("Should have set the status as not found for expired webhook")
-	}
+	_, err = cws.Get(h2.Id)
+	assert.NotNil(t, err, "Should have set the status as not found for expired webhook")
```
Comment (saturninoabril) [inline store/storetest/command_webhook_store.go]: I think all those `(not)nil` assertions to `err` should use `require` but will defer to other reviewers.

```suggestion
	require.NotNil(t, err, "Should have set the status as not found for expired webhook")
```

Comment (gupsho) [reply store/storetest/command_webhook_store.go]: Let me know and I can change to require.

Comment (mkraft) [reply store/storetest/command_webhook_store.go]: I prefer `require` myself, but I'm not opposed to `assert`.

Comment (DSchalla) [reply store/storetest/command_webhook_store.go]: Lets do `require` then.

Comment (gupsho) [reply store/storetest/command_webhook_store.go]: Done.

**Comment by gupsho** [comment]
> Aside from the comment from @saturninoabril this looks good to me, thanks for your PR 🎉

Changed.

**Comment by saturninoabril** [comment]
Thanks @gupsho!

**Pull request merged**